### ISSUE TITLE
Unable to run Snowstorm 7.6.0

### ISSUE THREAD

**Issue by hurtigcodes** [IHTSDO/snowstorm#366]
Hi, 

I'm unable to run the new snowstorm-7.6.0.jar. Could this issue have any relation to recent #340? We're not using a classification service in the server.

I've tried both the jar distributed here as well as my own jar built from source code using maven.

Logs:
```
2022-01-27 11:39:07.141  WARN 7644 --- [main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'remoteClassificationServiceClient': Unsatisfied dependency expressed through field 'classificationStatusService'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.snomed.snowstorm.core.data.services.classification.ClassificationStatusService' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
2022-01-27 11:39:07.145  INFO 7644 --- [IdentifierCacheManagerDaemon] o.s.s.c.d.s.i.IdentifierCacheManager     : Identifier cache manager sleep interrupted.
2022-01-27 11:39:07.145  INFO 7644 --- [IdentifierCacheManagerDaemon] o.s.s.c.d.s.i.IdentifierCacheManager     : Identifier cache manager polling stopped.
2022-01-27 11:39:07.154  INFO 7644 --- [main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2022-01-27 11:39:07.154 DEBUG 7644 --- [java-sdk-http-connection-reaper] com.amazonaws.http.IdleConnectionReaper  : Reaper thread: 

java.lang.InterruptedException: sleep interrupted
	at java.base/java.lang.Thread.sleep(Native Method)
	at com.amazonaws.http.IdleConnectionReaper.run(IdleConnectionReaper.java:181)

2022-01-27 11:39:07.155 DEBUG 7644 --- [java-sdk-http-connection-reaper] com.amazonaws.http.IdleConnectionReaper  : Shutting down reaper thread.
2022-01-27 11:39:07.177  INFO 7644 --- [main] ConditionEvaluationReportLoggingListener : 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2022-01-27 11:39:07.678 ERROR 7644 --- [main] o.s.b.d.LoggingFailureAnalysisReporter   : 

***************************
APPLICATION FAILED TO START
***************************

Description:

Field classificationStatusService in org.snomed.snowstorm.core.data.services.classification.RemoteClassificationServiceClient required a bean of type 'org.snomed.snowstorm.core.data.services.classification.ClassificationStatusService' that could not be found.

The injection point has the following annotations:
	- @org.springframework.beans.factory.annotation.Autowired(required=true)


Action:

Consider defining a bean of type 'org.snomed.snowstorm.core.data.services.classification.ClassificationStatusService' in your configuration.
```

**Comment by kaicode** [comment]
Hi there. You are right that this is related to #340. Snowstorm has default configuration to deal with this. The classification service is off by default. If the default properties are not being read you would have to provide your own in your external configuration.

You will experience this error if you are somehow not using the default configuration, for example if you use the `--spring.config.location` parameter on startup. That makes Spring Boot ignore all the defaults and only reads config from your config file. I strongly recommend _not_ to do this. We are careful when creating new configuration properties to set sensible defaults, for backward compatibility and stable behaviour. If your setup is not picking up the defaults you risk strange things happening and would have to update your config file every time you upgrade.

A better way is to use the defaults and only override the configuration properties you need to, using a configuration file that only contains config that needs to change in your setup. There are many ways to do this but the simplest is to have a file named `application.properties` in the same directory as the jar file when you start. Spring Boot will find that and combine the default configuration with the overrides in that file. There is no need to use `--spring.config.location` or any other parameter in that case. 

See [Spring Boot 2.x - Application Property Files](https://docs.spring.io/spring-boot/docs/2.3.3.RELEASE/reference/html/spring-boot-features.html#boot-features-external-config-application-property-files)

I hope that works for you!

**Comment by hurtigcodes** [comment]
Thanks again @kaicode, you were right.

I made the false assumption that the `application.properties` file in the src folder was loaded during app startup and that I needed to load a local `application.properties` to override values in the first file using the startup option  `--spring.config.location=/opt/snowstorm/application-local.properties`. 

Your recommendation (_"There are many ways to do this but the simplest is to have a file named application.properties in the same directory as the jar file when you start."_) worked perfectly, thanks!

**Comment by hurtigcodes** [comment]
Thanks again @kaicode, you were right.

I made the false assumption that the `application.properties` file in the src folder was loaded during app startup and that I needed to load a local `application.properties` to override values in the first file using the startup option  `--spring.config.location=/opt/snowstorm/application-local.properties`. 

Your recommendation (_"There are many ways to do this but the simplest is to have a file named application.properties in the same directory as the jar file when you start."_) worked perfectly, thanks!
